vdr/didweb/manager.go

```diff
@@ -67,8 +67,9 @@ type Manager struct {
 	keyStore crypto.KeyStore
 }
 
-func (m Manager) Deactivate(_ context.Context, _ did.DID) error {
-	return errors.New("Deactivate() is not yet supported for did:web")
+func (m Manager) Deactivate(_ context.Context, subjectDID did.DID) error {
+	// TODO: Should we remove private keys as well?
```

Comment:
Are there any risks if we keep them?

Reply:
it depends....
if keys are used for encryption then it becomes a choice between preventing stolen keys to be used to decrypt old data and allowing the owner to recover encrypted data at a later point in time.

Since we're only using keys for signing, I would say sanitise and delete, it's up to the secure storage to handle things like backups and recovery.

Reply:
Actually had to implement this throughout the crypto layer...